{"name": "test_TC_003_user_сan_autorize[locked_out_user-secret_sauce]", "status": "skipped", "statusDetails": {"message": "XFAIL\n\nAssertionError: wrong url", "trace": "self = <SauceDemoProject.tests.test_TC_003_user_can_be_autorize.TestSample object at 0x000002078445B8E0>\nd = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"a47f95410c0857a4c1011710c35403c5\")>\nusername = 'locked_out_user', password = 'secret_sauce'\n\n    @pytest.mark.parametrize(\n        \"username, password\",\n        [\n            (\"standard_user\", \"secret_sauce\"),\n            (\"performance_glitch_user\", \"secret_sauce\"),\n            (\"problem_user\", \"secret_sauce\"),\n            pytest.param(\n                \"locked_out_user\",\n                \"secret_sauce\",\n                marks=pytest.mark.xfail(raises=AssertionError),\n            ),\n        ],\n    )\n    def test_TC_003_user_сan_autorize(self, d, username, password):\n        page = LoginPage(d, link)\n        page.open_page()\n        page.register_user(username, password)\n        time.sleep(2)\n        page = ProductsPage(d, link)\n>       page.should_be_products_page()\n\ntest_TC_003_user_can_be_autorize.py:30: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\pages\\products_page.py:13: in should_be_products_page\n    self.should_be_link(ProductsPageSrc.LINK)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.products_page.ProductsPage object at 0x000002078455DF30>\nlink = 'inventory'\n\n    def should_be_link(self, link):\n>       assert link in self.browser.current_url, \"wrong url\"\nE       AssertionError: wrong url\n\n..\\pages\\base_page.py:59: AssertionError"}, "attachments": [{"name": "Screeshot", "source": "419fdd33-9363-4955-b615-839acd8e3b67-attachment.png", "type": "image/png"}, {"name": "stdout", "source": "c110d243-bfcf-456e-9834-d38133216e7a-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'locked_out_user'"}, {"name": "password", "value": "'secret_sauce'"}], "start": 1669800034538, "stop": 1669800036758, "uuid": "cea536af-e479-47bc-aca5-f06f52e8f3aa", "historyId": "14fec9e14c2eabd56f61c502cf39b536", "testCaseId": "58098719add8c8f75a04510e5eb59a4a", "fullName": "tests.test_TC_003_user_can_be_autorize.TestSample#test_TC_003_user_сan_autorize", "labels": [{"name": "tag", "value": "@pytest.mark.xfail(raises=<class 'AssertionError'>)"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_TC_003_user_can_be_autorize"}, {"name": "subSuite", "value": "TestSample"}, {"name": "host", "value": "DESKTOP-DBCHJ8G"}, {"name": "thread", "value": "20064-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_TC_003_user_can_be_autorize"}]}